app/templates/response/add_closing.js.html

```diff
@@ -19,7 +19,7 @@
             reason_ids.focus();
             $(this).prop('selected', !$(this).prop('selected'));
             return false;
-        });
+        });                       
```

Comment:
don't know how this made it through, pycharm's usually pretty good  with this type of thing...